### PR TITLE
Subscription Models, Part 1

### PR DESCRIPTION
**Pull request by chigby**
This pull request spins off the preliminary database setup from https://github.com/newamericafoundation/newamerica-cms/pull/1792 into its own branch. The purpose of this is to create new snippet models and page types that can be populated with the `make_subscribe_pages` command but won't affect the functionality of the site. This way we can set up the data beforehand and deploy the site changes in another release at a later date.

Order of operations:
- [ ] deploy #1808 (this branch)
- [ ] run `make_subscribe_pages`
- [ ] make subscribe page for the thread
- [ ] deploy #1792 
- [ ] run `clear_cache`
- [ ] update which are checked by default
- [ ] run `clear_cache` again

### CODE REVIEW

File: home/models.py [hunk 1]
```diff
@@ -485,7 +485,11 @@ class SubscribePage(OrgSimplePage):
     """
     Subscribe Page at the organization level
     """
-    parent_page_types = ['home.HomePage']
+    parent_page_types = [
```
Comment (nmorduch) [inline home/models.py:488]: Also needs to be able to go under the Thread homepage

Comment (chigby) [reply home/models.py:488]: Added this!

**Review by nmorduch — CHANGES_REQUESTED** [review]
Working great! One last change: Let's have all the new segments on the subscribe pages checked by default.

**Review by nmorduch — APPROVED** [review]
Perfect. Will wait to merge until the other is ready.

**Pull request merged**